webpack.config.js

```diff
@@ -20,6 +32,7 @@ module.exports = (env) => {
       path: path.resolve(__dirname, 'dist'),
       filename,
       library: 'PureCloudWebrtcSdk',
+      libraryExport: cdn ? 'PureCloudWebrtcSdk' : '',
```

Comment:
I had to do it this way to keep backwards compatibility with all other versions of 3.x. In v4, I may change it back.